examples/lib/tour/docsrc/01-getting-started/README.md

```diff
@@ -0,0 +1,198 @@
+# Basic Usage of OCM Repositories
+{{getting-started}}
+
+This [tour](example.go) illustrates the basic usage of the API to
+access component versions in an OCM repository.
+
+## Running the example
+
+You can just call the main program with some config file argument
+with the following content:
+
+```yaml
+component: github.com/mandelsoft/examples/cred1
+repository: ghcr.io/mandelsoft/ocm
+version: 0.1.0
+```
+
+## Walkthrough
+
+The basic entry point for using the OCM library is always
+an [OCM Context object](../../contexts.md). It bundles all
+configuration settings and type registrations, like
+access methods, repository types, etc, and
+configuration settings, like credentials,
+which should be used when working with the OCM
+ecosystem.
+
+Therefore, the first step is always to get access to such
+a context object. Our example uses the default context
+provided by the library, which covers the complete
+type registration contained in the executable.
+
+It can be accessed by a function of the ocm package.
+
+```go
+{{include}{../../01-getting-started/example.go}{default context}}
+```
+
+The context acts as the central entry
+point to get access to OCM elements.
+First, we get a repository, to look for
+component versions. We use the OCM
+repository providing the standard OCM
+components hosted on `ghcr.io`.
+
+For every storage technology used to store
+OCM components, there is a serializable
+descriptor object, the *repository specification*.
+It describes the information required to access
+the repository and can be used to store the serialized
+form as part of other resources, for example
+Kubernetes resources or configuration settings.
+The available repository implementations can be found
+under `.../pkg/contexts/ocm/repositories`.
+
+```go
+{{include}{../../01-getting-started/example.go}{repository spec}}
+```
+
+The context can now be used to map the descriptor
+into a repository object, which then provides access
+to the OCM elements stored in this repository.
+
+```go
+{{include}{../../01-getting-started/example.go}{repository}}
+```
+
+Many objects must be closed, if they should not be used
```

Comment:
```suggestion
To release potentially allocated temporary resources,
```

Reply:
This is the source (the other md file were generated :-)